crowdsourcing/serializers/project.py

```diff
@@ -36,12 +36,17 @@ class ModuleSerializer(DynamicFieldsModelSerializer):
     deleted = serializers.BooleanField(read_only=True)
     template = TemplateSerializer(many=True, read_only=False)
     module_tasks = TaskSerializer(many=True, read_only=True)
+    csv_data = serializers.ListField(write_only=True)
+    has_data_set = serializers.BooleanField(write_only=True)
```

Comment:
sorry still a bit confused. when i dont pass either parameter i start getting an error from the GET request to requester_projects when it tries to access the serialized data. does that mean i have to add csv_data to the module model?